x-pack/heartbeat/heartbeat.reference.yml

```diff
@@ -758,11 +758,11 @@ output.elasticsearch:
   # Resolve names locally when using a proxy server. Defaults to false.
   #proxy_use_local_resolver: false
 
-  # Enable SSL support. SSL is automatically enabled if any SSL setting is set.
+  # Use SSL settings for HTTPS.
```

Comment:
It appears that this file is out of date on master. These changes show up after `make update` on master too.

Reply:
Hm, let me check that and open another PR if we have some problems. This file should be fixed ASAP.

Reply:
This should be fixed by https://github.com/elastic/beats/pull/20681

Reply:
I have also updated this in https://github.com/elastic/beats/pull/20683, which I plan to merge as soon as the tests pass